src/NewTools-FileBrowser/String.extension.st

```diff
@@ -0,0 +1,12 @@
+Extension { #name : 'String' }
+
+{ #category : '*NewTools-FileBrowser' }
+String >> asFileName [
```

Comment:
Is it wanted to have this? I think this kind of methods were removed from Pharo not so long ago?

Reply:
I don't know, but I've checked and two external users use it:

- `Object>>#fuelOut`
- `CodeExporter class>>#writeSourceCodeFrom:baseName:isSt:`

![Captura de pantalla 2024-11-07 a las 15 52 47](https://github.com/user-attachments/assets/f4e811f6-ec95-46b9-8fb2-8b0c56e30863)

We can include it for now and open another issue to see if it is needed.